app-emulation/spice-protocol/spice_protocol-0.14.4.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="SPICE protocol definitions"
+DESCRIPTION="Headers defining the SPICE-protocol"
+HOMEPAGE="https://www.spice-space.org/"
+COPYRIGHT="2009 Red Hat, Inc"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://www.spice-space.org/download/releases/spice-protocol-$portVersion.tar.xz"
+CHECKSUM_SHA256="04ffba610d9fd441cfc47dfaa135d70096e60b1046d2119d8db2f8ea0d17d912"
+SOURCE_DIR="spice-protocol-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	spice_protocol = $portVersion
+	devel:spice_protocol = $portVersion
+"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:meson
+	cmd:ninja
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	rm -rf build
+
+	meson build --buildtype=release \
+		--prefix=$prefix --datadir=$developLibDir \
```

Comment:
The pkg-config file can reside in $dataDir/pkgconfig (didn't find it in $developLibDir here on 32bit)
```diff
-		--prefix=$prefix --datadir=$developLibDir \
+		--prefix=$prefix --datadir=$dataDir \
```

Reply:
1. witi **dataDir** specified, it will pack pc config in **/data/pkgconfig/spice-protocol.pc**
2. with **developLibDir** specified, it will pack pc config in **/develop/lib/pkgconfig/spice-protocol.pc**
According to the pack rules, we have to use 2 option. is there anyway to fix this for 32bit platform?

Reply:
We still ship packages that put the *.pc files in /data/pkgconfig, so it shouldn't be a problem there, haven't tested it if it resides in $libdir ...

Reply:
I don't have 32bit environment, it works?

Reply:
$dataDir is correct for architecture "any".

Reply:
I have checked xorgproto, it puts pkgconfig in **/data/pkgconfig**. maybe it should be puted where as it is.